lib/cv_theme.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:flutter/material.dart';
+
+class CVTheme {
+  CVTheme._();
+  static Color textFieldLabelColor(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? Colors.grey[300]
+        : Colors.grey[600];
+  }
+
+  static Color textColor(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? Colors.white
+        : Colors.black;
+  }
+
+  static Color primaryHeading(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? CVTheme.primaryColor
+        : Colors.black;
+  }
+
+  static Color boxBg(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? CVTheme.bgCardDark
+        : CVTheme.bgCard;
+  }
+
+  static Color boxShadow(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? CVTheme.bgCardDark
+        : CVTheme.grey;
+  }
+
+  static Color appBarText(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? CVTheme.primaryColor
+        : Colors.black;
+  }
+
+  static Color drawerIcon(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? CVTheme.primaryColor
+        : Colors.black;
+  }
+
+  static Color highlightText(context) {
+    return Theme.of(context).brightness == Brightness.dark
+        ? CVTheme.primaryColor
+        : CVTheme.primaryColorDark;
+  }
+
+  static ThemeData themeData = ThemeData.dark()
+      .copyWith(accentColor: Colors.green, primaryColor: CVTheme.primaryColor);
+
+  static const Color primaryColor = Color.fromRGBO(66, 185, 131, 1);
+  static const Color primaryColorDark = Color.fromRGBO(2, 110, 87, 1);
+  static const Color primaryColorLight = Color.fromRGBO(66, 185, 131, 0.5);
+  static const Color primaryColorShadow = Color.fromRGBO(245, 255, 252, 1);
+  static const Color imageBackground = Color.fromRGBO(63, 61, 86, 1);
+  static const Color secondaryColor = Color.fromRGBO(33, 33, 33, 1);
+  static const Color blue = Color.fromRGBO(66, 185, 182, 1);
+  static const Color red = Color.fromRGBO(255, 89, 89, 1);
+  static const Color grey = Color.fromRGBO(150, 150, 150, 1);
+  static const Color lightGrey = Color.fromRGBO(150, 150, 150, 0.5);
+  static const Color bgCard = Color.fromRGBO(255, 255, 255, 0.9);
+  static Color bgCardDark = Colors.grey[700];
```

Comment:
Mark these as const. Const has its own benefits and overall consistency matters.